backend/prompt_studio/prompt_profile_manager/profile_manager_helper.py

```diff
@@ -0,0 +1,15 @@
+from prompt_studio.prompt_profile_manager.models import ProfileManager
+
+
+class ProfileManagerHelper:
+
+    @classmethod
+    def get_profile_manager(cls, profile_manager_id: str) -> ProfileManager:
+        try:
+            return ProfileManager.objects.get(profile_id=profile_manager_id)
+        except ProfileManager.DoesNotExist:
+            raise ValueError("ProfileManager does not exist.")
+        except Exception as e:
+            raise RuntimeError(
+                f"An error occurred while retrieving ProfileManager: {e}"
+            )
```

Comment:
```suggestion
```
We are trying to catch and re-raise another exception here. This should be avoided at other places too if its followed. Instead we just let the exception propagate up the stack and it'll be caught at the caller of this function

Reply:
Got it